### PR TITLE
Feature/kp/lead stage card

### PR DESCRIPTION
**Pull request by kuldip-trentium**
dashboard admin api and UI created

### CODE REVIEW

File: app/(protected)/(tabs)/dashboard.tsx [hunk 1]
```diff
@@ -145,72 +154,82 @@ const Dashboard = () => {
...
+              <CompanyDashboardCard
+                leads={dashboardLeadList.leadStageCount?.map((item, index) => {
+                  return {
+                    title: item?.name,
+                    value: item?.leadCount,
+                    bgColor: chartColors[index],
+                    color: companyLeadTextColor[index],
+                  };
+                })}
+                name="Romes Smith"
```
Comment (Deeptrentium) [inline app/(protected)/(tabs)/dashboard.tsx:198]: why we use static name ?

File: components/molecules/CompanyDashboardCard/CompanyDashboardCard.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <LeadCount>{item.value}</LeadCount>
+      </LeadsCountContainer>
+    );
+  };
+
+  return (
+    <CompanyDashboardCardContainer>
+      <UserInformationContainer>
+        <ProfileIcon />
+        <UserName numberOfLines={1}>{name}</UserName>
```
Comment (Deeptrentium) [inline components/molecules/CompanyDashboardCard/CompanyDashboardCard.tsx:42]: UserNameText

File: components/organisms/LeadsProgressChart/LeadsProgressChart.styles.ts [hunk 3]
```diff
@@ -47,3 +46,17 @@ export const LabelContainer = styled(View)`
   justify-content: flex-start;
   height: 30px;
 `;
+export const NoLeadsProgressContainer = styled(View)<{ bgColor: string }>`
+  background-color: ${({ bgColor }) => bgColor};
+  border-radius: 43px;
+  height: 3px;
+  width: 70%;
```
Comment (Deeptrentium) [inline components/organisms/LeadsProgressChart/LeadsProgressChart.styles.ts:53]: please check

File: components/organisms/LeadsProgressChart/LeadsProgressChart.styles.ts [hunk 4]
```diff
@@ -47,3 +46,17 @@ export const LabelContainer = styled(View)`
...
+`;
+export const NoLeadsProgressView = styled(View)`
+  background-color: ${({ theme }) => theme.colors.white};
+  border-radius: 43px;
+  width: 80%;
```
Comment (Deeptrentium) [inline components/organisms/LeadsProgressChart/LeadsProgressChart.styles.ts:58]: please check

**Review by Deeptrentium — CHANGES_REQUESTED** [review]

File: app/(protected)/(tabs)/dashboard.tsx (hunk 1 above)
Comment (kuldip-trentium) [reply app/(protected)/(tabs)/dashboard.tsx:201]: bcz api integration is not done

Comment (Deeptrentium) [reply app/(protected)/(tabs)/dashboard.tsx:201]: then lets add a api call

File: components/organisms/LeadsProgressChart/LeadsProgressChart.styles.ts [hunk 5]
```diff
@@ -50,13 +50,15 @@ export const NoLeadsProgressContainer = styled(View)<{ bgColor: string }>`
   background-color: ${({ bgColor }) => bgColor};
   border-radius: 43px;
   height: 3px;
-  width: 70%;
+  width: 100%;
+  max-width: 70%;
```
Comment (Deeptrentium) [inline components/organisms/LeadsProgressChart/LeadsProgressChart.styles.ts:54]: width 100% and max width 70% what dose its means ?

File: app/(protected)/(tabs)/dashboard.tsx [hunk 6]
```diff
@@ -173,7 +173,7 @@ const Dashboard = () => {
                 leads={dashboardLeadList.leadStageCount?.map((item, index) => {
                   return {
                     label: item?.name,
-                    progress: item?.leadCount,
+                    progress: 0,
```
Comment (Deeptrentium) [inline app/(protected)/(tabs)/dashboard.tsx:176]: why 0 ?

**Review by Deeptrentium — CHANGES_REQUESTED** [review]

**Review by Deeptrentium — APPROVED** [review]

**Review by sagark1510 — APPROVED** [review]

**Pull request merged**